deno/send_message/src/functions/send_tweet.ts

```diff
@@ -0,0 +1,29 @@
+import { statusUpdate } from "https://kamekyame.github.io/twitter_api_client/api_v1/tweets/update.ts";
```

Comment:
So, I tried to build, but got an error:

```shell
$ docker run -e INTERNAL_RUNTIME_ENTRYPOINT=src/mod.ts --rm --interactive --tty --volume $PWD:/usr/code openruntimes/deno:v2-1.21 sh /usr/local/src/build.sh
Download https://deno.land/std@0.138.0/path/mod.ts
Download https://deno.land/x/oak@v10.5.1/mod.ts
Download https://deno.land/x/oak@v10.5.1/application.ts
Download https://deno.land/x/oak@v10.5.1/body.ts
Download https://deno.land/x/oak@v10.5.1/context.ts
Download https://deno.land/x/oak@v10.5.1/cookies.ts
Download https://deno.land/x/oak@v10.5.1/deps.ts
Download https://deno.land/x/oak@v10.5.1/etag.ts
Download https://deno.land/x/oak@v10.5.1/helpers.ts
Download https://deno.land/x/oak@v10.5.1/httpError.ts
Download https://deno.land/x/oak@v10.5.1/http_server_native.ts
Download https://deno.land/x/oak@v10.5.1/http_server_native_request.ts
Download https://deno.land/x/oak@v10.5.1/middleware.ts
Download https://deno.land/x/oak@v10.5.1/middleware/proxy.ts
Download https://deno.land/x/oak@v10.5.1/multipart.ts
Download https://deno.land/x/oak@v10.5.1/range.ts
Download https://deno.land/x/oak@v10.5.1/request.ts
Download https://deno.land/x/oak@v10.5.1/response.ts
Download https://deno.land/x/oak@v10.5.1/router.ts
Download https://deno.land/x/oak@v10.5.1/send.ts
Download https://deno.land/x/oak@v10.5.1/server_sent_event.ts
Download https://deno.land/x/oak@v10.5.1/testing.ts
Download https://deno.land/x/oak@v10.5.1/types.d.ts
Download https://deno.land/x/oak@v10.5.1/util.ts
Download https://deno.land/std@0.138.0/_util/os.ts
Download https://deno.land/std@0.138.0/path/_interface.ts
Download https://deno.land/std@0.138.0/path/common.ts
Download https://deno.land/std@0.138.0/path/glob.ts
Download https://deno.land/std@0.138.0/path/posix.ts
Download https://deno.land/std@0.138.0/path/separator.ts
Download https://deno.land/std@0.138.0/path/win32.ts
Download https://deno.land/x/oak@v10.5.1/keyStack.ts
Download https://deno.land/x/oak@v10.5.1/structured_clone.ts
Download https://deno.land/x/oak@v10.5.1/isMediaType.ts
Download https://deno.land/std@0.131.0/bytes/mod.ts
Download https://deno.land/std@0.131.0/encoding/base64.ts
Download https://deno.land/std@0.131.0/http/http_status.ts
Download https://deno.land/std@0.131.0/io/readers.ts
Download https://deno.land/std@0.131.0/path/mod.ts
Download https://deno.land/std@0.131.0/streams/conversion.ts
Download https://deno.land/x/media_types@v3.0.2/mod.ts
Download https://deno.land/x/path_to_regexp@v6.2.0/index.ts
Download https://deno.land/x/oak@v10.5.1/negotiation/charset.ts
Download https://deno.land/x/oak@v10.5.1/negotiation/encoding.ts
Download https://deno.land/x/oak@v10.5.1/negotiation/language.ts
Download https://deno.land/x/oak@v10.5.1/negotiation/mediaType.ts
Download https://deno.land/x/oak@v10.5.1/buf_reader.ts
Download https://deno.land/x/oak@v10.5.1/content_disposition.ts
Download https://deno.land/x/oak@v10.5.1/headers.ts
Download https://deno.land/std@0.138.0/path/_constants.ts
Download https://deno.land/std@0.138.0/path/_util.ts
Download https://deno.land/x/oak@v10.5.1/tssCompare.ts
Download https://deno.land/x/oak@v10.5.1/mediaTyper.ts
Download https://deno.land/std@0.138.0/_util/assert.ts
Download https://deno.land/std@0.131.0/bytes/equals.ts
Download https://deno.land/std@0.131.0/io/buffer.ts
Download https://deno.land/std@0.131.0/_util/os.ts
Download https://deno.land/std@0.131.0/path/_interface.ts
Download https://deno.land/std@0.131.0/path/common.ts
Download https://deno.land/std@0.131.0/path/glob.ts
Download https://deno.land/std@0.131.0/path/posix.ts
Download https://deno.land/std@0.131.0/path/separator.ts
Download https://deno.land/std@0.131.0/path/win32.ts
Download https://raw.githubusercontent.com/jshttp/mime-db/v1.52.0/db.json
Download https://deno.land/x/oak@v10.5.1/negotiation/common.ts
Download https://deno.land/std@0.131.0/_util/assert.ts
Download https://deno.land/std@0.131.0/bytes/bytes_list.ts
Download https://deno.land/std@0.131.0/io/types.d.ts
Download https://deno.land/std@0.131.0/path/_constants.ts
Download https://deno.land/std@0.131.0/path/_util.ts
Check file:///usr/local/src/server.ts
Download https://kamekyame.github.io/twitter_api_client/api_v1/tweets/update.ts
error: Module not found "https://kamekyame.github.io/twitter_api_client/api_v1/tweets/update.ts".
    at file:///usr/builds/src/functions/send_tweet.ts:1:30
```

Looks like this is the right import.

```suggestion
import { statusUpdate } from "https://deno.land/x/twitter_api_client/api_v1/tweets/update.ts";
```